test-fork/peripheral/grants/AuraArbBalGrant.spec.ts

```diff
@@ -0,0 +1,276 @@
+import { expect } from "chai";
+import { BigNumber, BigNumberish, ethers } from "ethers";
+import hre, { network } from "hardhat";
+
+import {
+    Account,
+    AuraArbBalGrant,
+    AuraArbBalGrant__factory,
+    IBalancerPool,
+    IBalancerPool__factory,
+    IBalancerVault__factory,
+    IERC20,
+    IERC20__factory,
+    IWeightedPoolFactoryV2__factory,
+} from "../../../types";
+import { config } from "../../../tasks/deploy/mainnet-config";
+import { getPoolAddress, Phase2Deployed } from "../../../scripts/deploySystem";
+import { impersonateAccount, ZERO_ADDRESS, simpleToExactAmount } from "../../../test-utils";
+import { AssetHelpers } from "@balancer-labs/balancer-js";
+
+const FORK_BLOCK = 17274000;
+const ARB_TOKEN = "0xB50721BCf8d664c30412Cfbc6cf7a15145234ad1";
+
+interface DeployedPool {
+    address: string;
+    poolTokens: [string, string, string];
+    weights: [BigNumber, BigNumber, BigNumber];
+    tokenOrder: [number, number, number];
+}
+
+describe("AuraArbBalGrant", () => {
+    let grant: AuraArbBalGrant;
+    let balToken: IERC20;
+    let phase2: Phase2Deployed;
+
+    let deployer: Account;
+    let balancer: Account;
+    let project: Account;
+    let random: Account;
+
+    let pool: IBalancerPool;
+    let deployedPool: DeployedPool;
+
+    const deployPool = async (): Promise<DeployedPool> => {
+        const balHelper = new AssetHelpers(config.addresses.weth);
+
+        const poolFactory = IWeightedPoolFactoryV2__factory.connect(
+            config.addresses.balancerPoolFactories.weightedPool,
+            deployer.signer,
+        );
+
+        const tokens = [ARB_TOKEN, phase2.cvx.address, config.addresses.token];
+
+        const [poolTokens, weights] = balHelper.sortTokens(tokens, [
+            simpleToExactAmount(34, 16),
+            simpleToExactAmount(33, 16),
+            simpleToExactAmount(33, 16),
```

Comment:
Nit: We could make these way more granular, 33.333, 33.333, 33.334, instead of one token having a whole 1% extra :P